### PR TITLE
Replace docker with podman

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,24 +24,25 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Build the image
-        run: "cekit --verbose build --overrides \"{'labels': [{'name': 'quay.expires-after', 'value': '1w'}, {'name': 'git-sha', 'value': '$GITHUB_SHA'}]}\" docker --tag $IMAGE_NAME:latest"
+        run: >
+          cekit --verbose build --overrides "{'labels': [{'name': 'quay.expires-after', 'value': '1w'}, {'name': 'git-sha', 'value': '$GITHUB_SHA'}]}" docker --tag $IMAGE_NAME:latest
```
Comment (lenosi) [inline .github/workflows/ci.yml]: Remains docker, `cekit build podman`

Comment (lenosi) [reply .github/workflows/ci.yml]: And you will need to add a source for get podman on ubuntu. I still not much understand why we are replacing docker cli on ubuntu machine in GH Actions with docker but anyway :sweat_smile: 

```
echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key | sudo apt-key add -
sudo apt-get update
sudo apt-get -y upgrade 
sudo apt-get -y install podman
```

Comment (brusdev) [reply .github/workflows/ci.yml]: Podman is already available on GitHub Actions Containers: https://github.com/actions/virtual-environments/issues/320, we don't need to install it 😅 🎉

Comment (brusdev) [reply .github/workflows/ci.yml]: I have pushed another commit to remove the latest references to docker

**Comment by rhmessagingqe** [comment]
Some Test Comment

**Comment by rhmessagingqe** [comment]
Some Test Comment

**Comment by tlbueno** [comment]
I am closing this as already done in comit https://github.com/artemiscloud/activemq-artemis-broker-image/commit/9629fd42c0d14542d461fe90d0d11be4663219e2

**Pull request closed without merging**